### PR TITLE
Allow create to skip existing ingresses.

### PR DESCRIPTION
**Pull request by perotinus**
@nikhiljindal @madhusudancs @G-Harmon @csbell

### CODE REVIEW

**Comment by nikhiljindal** [comment]
haha silly mistake :)
@csbell  is going to be very happy with this change!

/lgtm

File: app/kubemci/cmd/create.go [hunk 1]
```diff
@@ -206,11 +206,10 @@ func createIngressInClusters(kubeconfig, ingressFilename string, clusters []stri
 		actualIng, createErr := client.Extensions().Ingresses(ing.Namespace).Create(&ing)
 		glog.V(2).Infof("Ingress Create returned: err:%v. Actual Ingress:%+v", err, actualIng)
```
Comment (G-Harmon) [inline app/kubemci/cmd/create.go]: also s/err/createErr/ here

**Comment by G-Harmon** [comment]
d'oh! my bad.

File: app/kubemci/cmd/create.go (hunk 1 above)
Comment (perotinus) [reply app/kubemci/cmd/create.go]: Done in #37.

**Pull request merged**